web/src/common/ProjectStorage.ts

```diff
@@ -234,9 +234,9 @@ export class ProjectStorage {
   }
 
   nextAvailableNeighbourhoodName(projectID: ProjectID): string {
-    let project = this.project(projectID);
-    let basename = `${project.project_name} LTN`;
+    let basename = "My neighbourhood";
```

Comment:
Is "My" too corny?

As I said in the description, with the project name in the breadcrumbs, also including it here felt redundant.

But just calling it "LTN" felt a bit sparse. I also don't know if that abbreviation is meaningful enough to everyone who might use the app.

Anyway, open to suggestions here!

Reply:
This seems fine to me. We can get feedback this next week on it.